src/qa-test-eks.sh

```diff
@@ -1,15 +1,31 @@
 #!/bin/bash
-set -eux -o pipefail
+set -eux #-o pipefail
 
 BASEPATH=./test/integration
 ACTION=$1
 # $AWS_DEFAULT_REGION
 
-function extra_cleanup {
+function cleanup_cluster {
 
     # Remove specific resources that sometimes get left behind (always return true, as resource may have been successfully been cleaned up)
-    aws --region "$REGION" iam list-roles --output json | jq -r --arg ROLEPREFIX "eks-${CLUSTERNAME}-" '.Roles[] | select( .RoleName | contains($ROLEPREFIX, "Velero") ) | .RoleName' | xargs -r -L1 aws --region "$REGION" iam delete-role --role-name || true
-    aws --region "$REGION" ec2 describe-network-interfaces --filters "Name=group-name,Values=eks-${CLUSTERNAME}-node" --query "NetworkInterfaces[].NetworkInterfaceId" --output text | xargs -r -L1 aws --region "$REGION" ec2 delete-network-interface --network-interface-id || true
+
+    # Detach any policies from the EKS node role
+    aws --region "$REGION" iam list-attached-role-policies --role-name "eks-${CLUSTERNAME}-node" --output json | jq '.AttachedPolicies[].PolicyArn' | xargs -tr -L1 aws --no-cli-pager iam detach-role-policy --role-name "eks-${CLUSTERNAME}-node" --policy-arn || true
```

Comment:
Shouldn't we be looping through all the eks-clustername roles it finds and detaching policies and instance profiles from them, rather than being specific to just policies on eks-node? It'd potentially catch more